### ISSUE TITLE
404 URL for 'Accessible Name Calculation' anchor

### ISSUE THREAD

**Issue by boutrosabichedid** [w3c/wai-wcag-quickref#157]
_Accessible Name Calculation_ anchor link is broken (404).
This applies to wherever it exists in this [WCAG20 - TECHS document](https://www.w3.org/TR/WCAG20-TECHS/) or [WCAG20 quickref document](https://www.w3.org/WAI/WCAG20/quickref/).

**Comment by yatil** [comment]
Hi @boutrosabichedid, can you tell me what you searched for? I couldn’t find the wrong link with a quick search. Thanks!

**Comment by boutrosabichedid** [comment]
Hi @yatil:
I did not search for it specifically, but I was reading several accessibility documents. Here is an example, but I know the 404 Link is used in many other places, specifically in the secondary links 
 (General Techniques, sufficient Techniques, ...) associated with the documents I mentioned in my comment above.

One example I found from a direct Google search:
Search 'Accessible Name Calculation' in the following [document](https://www.w3.org/TR/html-aapi/)

The 404 URL associated with the anchor text is: https://www.w3.org/TR/wai-aria/roles#namecalculation
Thanks.
